release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,389 @@
+
+//ROX-27011
+[id="enhanced-visibility-for-the-openshift-component-cves_{context}"]
+=== Enhanced visibility for the OpenShift Container Platform component CVEs
+
+{product-title-short} 4.7 introduces improved visibility for the {ocp} component Common Vulnerabilities and Exposures (CVEs). You can now view these CVEs separately from the *Platform Components* -> *User Workload CVEs* page to assess and manage security risks.
+
+//ROX-26781
+[id="machine-to-machine-authentication-by-using-microsoft-entra-id_{context}"]
+=== Machine-to-Machine authentication by using Microsoft Entra ID
+
+In {product-title-short} 4.7, you can now enable machine-to-machine authentication by using the short-lived {product-title-short} Central tokens with Microsoft Entra ID, formerly known as Azure AD, as the OpenID Connect (OIDC) identity provider for your organization. 
+You can now exchange ID tokens issued to your application's service principals for {product-title-short} Central access tokens, which you can then use to authenticate {product-title-short} APIs.
+
+For more information, see xref:../operating/manage-user-access/configure-short-lived-access.adoc#configure-short-lived-access[Configuring short-lived access].
+
+//ROX-26546
+[id="view-the-violation-status-directly-on-the-violations-page_{context}"]
+=== View the violation status directly on the Violations page
+
+With {product-title-short} 4.7, you can now see the status of a violation directly on the *Violations* page, and determine if the violation has been resolved. This enhances automation, as the links to the alerts now show the full context when they are accessed.
+
+//ROX-26335
+[id="prioritize-cves-with-the-epss-integration_{context}"]
+=== Prioritize CVEs with the EPSS integration
+
+{product-title-short} 4.7 introduces integration with the Exploit Prediction Scoring System (EPSS), a data-driven model that estimates the likelihood of a software vulnerability being exploited. 
+
+In addition to the severity and Common Vulnerability Scoring System (CVSS) score, an EPSS probability score from 0%-100% is now displayed for each detected Common Vulnerabilities and Exposures (CVEs). 
+You can use the EPSS score to better prioritize the remediation of CVE vulnerabilities and strengthen your security strategy.
+
+//ROX-26330
+[id="gain-visibility-into-the-external-ips-in-the-network-graph_{context}"]
+=== Enhanced visibility into the external IPs in the network graph
+
+With {product-title-short} 4.7, you can now get a better insight into the external IPs behind external entities in the network graph. 
+This enhancement helps you identify the specific IP addresses your deployments are communicating with and gives you better insight into the external interactions and potential risks.
+This feature was a technology preview in {product-title-short} 4.6.0 and is now generally available in {product-title-short} 4.7.0.
```

Comment:
The external IPs feature is not GA in 4.7. cc @Stringy @ovalenti

Reply:
Yes, still TP in the docs also.